docs/setup-aws-lambda.asciidoc

```diff
@@ -49,41 +39,42 @@ explicitly.
-section of the extension documentation need to be configured. The Java agent utilizes a https://docs.aws.amazon.com/lambda/latest/dg/runtimes-modify.html[wrapper script]
-to set some of these up, so follow the next section to see which are required.
+Follow the https://www.elastic.co/guide/en/apm/guide/current/aws-lambda-extension.html[installation documentation for the extension] to setup the Elastic APM Lambda extension for your Lambda function.
 
 [float]
 [[aws-lambda-instrumenting]]
-==== Installing the Elastic APM Java Agent layer
-
-The Java Agent is installed as an AWS Layer as well. It is distributed as a ZIP archive containing two files:
-
-- The Java Agent jar file
-- A wrapper script that sets up the runtime to enable automatic attachment of the agent
-
-Follow these steps to ensure proper Java Agent attachment to your Lambda function:
-
-1. Download the `elastic-apm-java-aws-lambda-layer-<VERSION>.zip` archive from our
-https://github.com/elastic/apm-agent-java/releases[GitHub Releases page].
-2. Publish the downloaded ZIP archive as an https://docs.aws.amazon.com/lambda/latest/dg/invocation-layers.html?icmpid=docs_lambda_help[AWS Lambda Layer].
-3. Configure your function to use the Java Agent layer.
-4. Within your Function configuration, configure the following environment variables:
-a. `AWS_LAMBDA_EXEC_WRAPPER` (required): set this variable's value to `/opt/elastic-apm-handler`, so that Lambda starts the runtime with this script.
-b. `ELASTIC_APM_LAMBDA_APM_SERVER` (required): this will tell the Elastic APM Extension where to send data to.
-c. <<config-secret-token, `ELASTIC_APM_SECRET_TOKEN`>> or <<config-api-key, `ELASTIC_APM_API_KEY`>> (required): one of these needs to be set
-as the authentication method that the extension uses when sending data to the URL configured via `ELASTIC_APM_LAMBDA_APM_SERVER`
-d. <<config-application-packages, `ELASTIC_APM_APPLICATION_PACKAGES`>> (recommended): setting this may improve cold start times.
-5. Fine-tune the Java agent with any of the available <<configuration, configuration options>>.
+==== Install the Elastic APM Java Agent Layer
+
+The Java Agent is installed as an AWS Layer. 
+
+1. Pick the right ARN from the https://github.com/elastic/apm-agent-java/releases[ARN table for the APM Java Agent Lambda Layer]. Note: The APM Java Agent Layer needs to be in the _same region_ as your Lambda function.
+2. Navigate to your function in the AWS Console
+3. Scroll to the Layers section and click the _Add a layer_ button
+4. Choose the _Specify an ARN_ radio button
+5. Enter the Version ARN of the APM Java Agent Layer in the _Specify an ARN_ text input
+6. Click the _Add_ button
+
+Once the APM Java Agent Layer is in place, you'll need to configure some environment variables _in addition_ to the environment variables you already configured for the APM Lambda extension. 
+Configure the following environment variables for the APM Java Agent Layer:
```

Comment:
I don't think it is better than what we had before:
```
NOTE: Not all environment variables specified within The Necessary Variables
section of the extension documentation need to be configured. The Java agent 
utilizes a wrapper script to set some of these up, so follow the next section to 
see which are required.
```

The AWS Lambda extension [docs](https://www.elastic.co/guide/en/apm/guide/current/aws-lambda-extension.html#aws-lambda-variables) include env variables that are not required for Java. 
This section even starts with a note:
```
The necessary environment variables depend on the APM agent being used. Follow the 
Node.js agent setup guide, Python agent setup guide or Java agent setup guide, r
espectively, for specific instructions on setting the environment variables.
```
But now we don't list the minimal required env variables

Reply:
I just saw https://github.com/elastic/apm-aws-lambda/pull/122 (the online docs don't include it yet).
So I guess we moved this entire section about - not all agents require the same variables?
What is telling our users that the layer script sets some of the env variables but not others?

Reply:
Yes, the extension docs only contain the extension-specific configs now. And all of the agents are expected to set meaningful default for most of the config options, so no need to document it there. 

With that we can focus in the Java docs on the Java-specific env vars.

Reply:
OK, I reviewed in context of the updated extension docs, makes sense. Thanks